src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -2925,6 +2969,135 @@ void ProcessSystemTypeGetMemberLikeCall (System.Reflection.MemberTypes memberTyp
 					break;
 				}
 			}
+
+			//
+			// arity == null for name match regardless of arity
+			//
+			void MarkMethodsFromReflectionCall (
```

Comment:
Existing code style does not use C-like formatting for declaration, please follow the existing code style.

Reply:
Should be fixed now